From cbf6303b225a3fc9fd505538416336e1a2bc5a0e Mon Sep 17 00:00:00 2001
From: Dingqiang Lin <jon.lin@rock-chips.com>
Date: Wed, 13 Jun 2018 17:56:21 +0800
Subject: [PATCH] arm64: configs: rk3308_linux_defconfig: enable slc nand and
 sfc devices driver

1.Disable former nandc configs and change to sftl slc nand;
2.Enables sfc spi nand drivers and sfc spi nand drivers.

Change-Id: Ia17e4e5cac5a2859cf2cb2013d40c446b88194b5
Signed-off-by: Dingqiang Lin <jon.lin@rock-chips.com>
---
 arch/arm64/configs/rk3308_linux_defconfig | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/configs/rk3308_linux_defconfig b/arch/arm64/configs/rk3308_linux_defconfig
index f9c5775f2154..a1555cb1479f 100644
--- a/arch/arm64/configs/rk3308_linux_defconfig
+++ b/arch/arm64/configs/rk3308_linux_defconfig
@@ -229,7 +229,10 @@ CONFIG_PHY_ROCKCHIP_INNO_USB2=y
 CONFIG_ANDROID=y
 CONFIG_NVMEM=y
 CONFIG_ROCKCHIP_OTP=y
-CONFIG_RK_NAND=y
+CONFIG_RK_FLASH=y
+CONFIG_RK_NANDC_NAND=y
+CONFIG_RK_SFC_NAND=y
+CONFIG_RK_SFC_NOR=y
 CONFIG_ROCKCHIP_SIP=y
 CONFIG_EXT2_FS=y
 CONFIG_EXT4_FS=m
-- 
2.35.3

